thorlcr/master/thmastermain.cpp

```diff
@@ -949,14 +971,20 @@ int main( int argc, const char *argv[]  )
                 numWorkers = globals->getPropInt("@numWorkers", 0);
             if (0 == numWorkers)
                 throw makeStringException(0, "Number of workers must be > 0 (numWorkers)");
+            if (wuRead->hasDebugValue("numWorkersPerPod"))
+                numWorkersPerPod = wuRead->getDebugValueInt("numWorkersPerPod", 0);
+            else
+                numWorkersPerPod = globals->getPropInt("@numWorkersPerPod", 1); // default to 1
+            if ((numWorkers % numWorkersPerPod) != 0)
```

Comment:
Danger of divide by zero here?

Reply:
the values schema should check it (min value: 1), but could be overriden as a workunit property (to 0), so I'll add a check/throw an meaingful exception here if invalid.